src/index.js

```diff
@@ -1,28 +1,153 @@
-import './index.css';
+/* eslint-disable no-use-before-define */
 
-const tasks = [
-  { description: 'Work on a project', completed: true, index: 0 },
-  { description: 'Read a novel', completed: true, index: 1 },
-  { description: 'Learn Webpack', completed: false, index: 2 },
-  { description: 'Do House chores', completed: false, index: 3 },
-];
+import './index.css';
+import completeStatus from './status.js';
 
+let tasks = [];
+if (localStorage.getItem('tasks')) {
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+}
```

Comment:
- [ ] Please make sure there is no error when the project start and when adding a task. 
**Your project error**
![image](https://user-images.githubusercontent.com/74411135/236568453-149a4b6d-d0f8-4bf0-8aa5-4cb61dd8e96b.png)

- [ ] Please consider making sure the request implementation is also done:

> * [ ]  Good job so far 👏 It looks like the suggestion from the previous reviewer has not been implemented yet, as he said when you try to check the checkbox to mark the task as completed FOR THE FIRST TIME, It shows the error which is shown below 👇
> 
> ![image](https://user-images.githubusercontent.com/90222110/236560444-2d78d012-170b-418d-8be0-c5c2a1906bd2.png)
> 
> please debug your function in order to collect that. I saw that after I try to remove that error screen, the functionality works so I suggest that you correct that compilation error 😉

 **Note that I cannot check that as I cannot add any stack, I can't also check them**. 

This will improve the user experience 😄 📱